neonvm/tools/vm-builder/main.go

```diff
@@ -250,24 +258,18 @@ fi
 ::respawn:su -p nobody -c '/usr/local/bin/pgbouncer /etc/pgbouncer.ini'
 ::respawn:su -p nobody -c 'DATA_SOURCE_NAME="user=cloud_admin sslmode=disable dbname=postgres" /bin/postgres_exporter --auto-discover-databases --exclude-databases=template0,template1'
 ttyS0::respawn:/neonvm/bin/agetty --8bits --local-line --noissue --noclear --noreset --host console --login-program /neonvm/bin/login --login-pause --autologin root 115200 ttyS0 linux
+::shutdown:/neonvm/bin/vmshutdown
```

Comment:
does `::shutdown` actually exist? I couldn't find it, but maybe I was looking at the wrong man page :sweat_smile:

Reply:
Yes it does, the online documentation is terrible. I think it's been around in several init implementation, and I know it's present in the busybox init.
See the source code links I added to the PR description.